### DISCUSSION TITLE
filterOption filter not assigned child

### DISCUSSION THREAD

**Discussion by MattRvt** [payloadcms/payload#737] [Q&A]
Hello,
I'm getting stuck and need help 😅

I have a parent entity with a child field. 

On admin UI, I want to display only the unassigned child.

For example:

| key | value |
|--|--|
| parent1 | [child1,child2]  |
| | child3|
||child4|

I want to display only child3 and child4.

In SQL, this would be as easy as 
``` sql
 select * from child where parent_id is null
 ```

I'm sorry if the question is too simple, I'm quite new to MongoDB, and it's environment.

Thank you for taking the time to read me 😁

**Comment by DanRibbens** [comment]
Hey @MattRvt,

Sometimes I like to go to the list view in the admin UI to test what I need to make for `filterOptions`. Go to the list view for your child collection you will see filter options in the admin UI. Add a filter for `parent`,  `equals` `null` and you should be able to see your results. The network tab will show what the url params are in the `get` request. Does that have the results you want?

If so, then your query in the `filterOptions` will look like:
```
where: {
  parent: { equals: null },
}
```

Good luck to you!

**Reply by MattRvt** [reply]
I tried this but I still get all the record.

To be more specific, I have a plane model that has many maintenances to do. And I don't want to get all the maintenaces of the DB when admin ads a model.

so, as you suggest, I tried this:
``` TS
{
    name: 'scheduledMaintenances',
    type: 'relationship',
    relationTo: 'scheduled-maintenances',
    hasMany: true,
    filterOptions: ({ relationTo, data, siblingData, id, user }) => {
        // maintenance that has no model
        return {
            models: { equals: null },
        }
    }
}
```
Also, I expected to get an error, as I think maintenance has no field model.

Anyway, thanks for your help, I appreciate it.
